frontend/tsconfig.json

```diff
@@ -18,7 +18,7 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react-jsx"
+    "jsx": "react"
```

Comment:
whats the reason for this change?

Reply:
So this is actually a change I told her to make locally to make vscode red lines go away. @r-aya don't push this change to github - just do it locally to remove red underlines